app/build.gradle

```diff
@@ -18,6 +18,7 @@ repositories {
 dependencies {
     // Use JUnit Jupiter for testing.
     testImplementation 'org.junit.jupiter:junit-jupiter:5.7.1'
+    testImplementation 'org.mockito:mockito-core:5.5.0'
```

Comment:
You can use mockito for mocking out dependencies of the services you are trying to test.
E.g. I needed to mock the queue's dequeueJob in my dispatcher test because I'm not trying to test the queue, only the dispatcher.